types/stacktrace-js/index.d.ts

```diff
@@ -47,6 +48,13 @@ declare namespace StackTrace {
   export function get(options?: StackTraceOptions): Promise<StackFrame[]>;
 
   /**
+   * Get a backtrace from invocation point synchronously.
+   * @param options Options Object
+   * @return Array[StackFrame]
+   */
+  export function getSync(options?: { filter?: (stackFrame: StackFrame) => boolean }): StackFrame[]
```

Comment:
I didn't realise possibility of inconsistency in the pkg, I only see those specs defined in README, but missed the .d.ts file 😂.